thunder/benchmarks/benchmark_litgpt.py

```diff
@@ -188,6 +224,8 @@ def __init__(
         self.model = self.init_model()
         print(f"Time to instantiate model: {time.perf_counter() - t0:.02f} seconds.")
 
+        self.model = get_converted_model(low_precision_mode, self.compile, self.model)
```

Comment:
here too we could be more explicit: just condition of `if use_transformer_engine` (as defined in the other comment) and convert the model, and leave it alone in the else

Reply:
Done